2023-04-13 17:17:29,065 INFO root Token successfully initted: 6234331500:AAG2Hcuep4y4N58uuJeXrHnjbpaSyrpXk3c
2023-04-13 17:17:29,065 INFO root Bot successfully initted
2023-04-13 17:17:29,066 INFO root DB connected successfully, cursor created
2023-04-13 17:17:29,069 DEBUG urllib3.connectionpool Starting new HTTPS connection (1): api.telegram.org:443
2023-04-13 17:17:29,389 DEBUG urllib3.connectionpool https://api.telegram.org:443 "GET /bot6234331500:AAG2Hcuep4y4N58uuJeXrHnjbpaSyrpXk3c/getMe HTTP/1.1" 200 191
2023-04-13 17:17:29,391 DEBUG urllib3.connectionpool Starting new HTTPS connection (1): api.telegram.org:443
2023-04-13 17:17:32,897 DEBUG urllib3.connectionpool https://api.telegram.org:443 "GET /bot6234331500:AAG2Hcuep4y4N58uuJeXrHnjbpaSyrpXk3c/getUpdates?offset=1&timeout=20 HTTP/1.1" 200 398
2023-04-13 17:17:32,901 DEBUG urllib3.connectionpool Starting new HTTPS connection (1): api.telegram.org:443
2023-04-13 17:17:33,196 DEBUG urllib3.connectionpool https://api.telegram.org:443 "POST /bot6234331500:AAG2Hcuep4y4N58uuJeXrHnjbpaSyrpXk3c/sendMessage?chat_id=1515397583&text=%D0%9F%D0%BE%D0%B6%D0%B0%D0%BB%D1%83%D0%B9%D1%81%D1%82%D0%B0%2C+%D0%B2%D0%B2%D0%B5%D0%B4%D0%B8%D1%82%D0%B5+%D0%92%D0%B0%D1%88+%D0%B0%D0%BD%D0%B5%D0%BA%D0%B4%D0%BE%D1%82.+%D0%95%D1%81%D0%BB%D0%B8+%D0%BF%D0%B5%D1%80%D0%B5%D0%B4%D1%83%D0%BC%D0%B0%D0%BB%D0%B8+-+%2Fcancel+%D0%B2+%D0%BF%D0%BE%D0%BC%D0%BE%D1%89%D1%8C&reply_markup=%7B%22inline_keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22%5Cu041e%5Cu0442%5Cu043c%5Cu0435%5Cu043d%5Cu0430%22%7D%5D%5D%7D HTTP/1.1" 400 144
2023-04-13 17:17:33,200 ERROR TeleBot Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 400. Description: Bad Request: can't parse inline keyboard button: Text buttons are unallowed in the inline keyboard
2023-04-13 17:17:33,201 ERROR TeleBot Exception traceback:
Traceback (most recent call last):
  File "C:\Users\яндекс 2 год (3)\AppData\Local\Programs\Python\Python39\lib\site-packages\telebot\__init__.py", line 1074, in __threaded_polling
    self.worker_pool.raise_exceptions()
  File "C:\Users\яндекс 2 год (3)\AppData\Local\Programs\Python\Python39\lib\site-packages\telebot\util.py", line 148, in raise_exceptions
    raise self.exception_info
  File "C:\Users\яндекс 2 год (3)\AppData\Local\Programs\Python\Python39\lib\site-packages\telebot\util.py", line 91, in run
    task(*args, **kwargs)
  File "C:\Users\яндекс 2 год (3)\AppData\Local\Programs\Python\Python39\lib\site-packages\telebot\__init__.py", line 6428, in _run_middlewares_and_handler
    result = handler['function'](message)
  File "C:\favanek\main.py", line 86, in suggest_message
    bot.send_message(message.chat.id, 'ѕожалуйста, введите ¬аш анекдот. ≈сли передумали - /cancel в помощь',
  File "C:\Users\яндекс 2 год (3)\AppData\Local\Programs\Python\Python39\lib\site-packages\telebot\__init__.py", line 1549, in send_message
    apihelper.send_message(
  File "C:\Users\яндекс 2 год (3)\AppData\Local\Programs\Python\Python39\lib\site-packages\telebot\apihelper.py", line 264, in send_message
    return _make_request(token, method_url, params=payload, method='post')
  File "C:\Users\яндекс 2 год (3)\AppData\Local\Programs\Python\Python39\lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
  File "C:\Users\яндекс 2 год (3)\AppData\Local\Programs\Python\Python39\lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 400. Description: Bad Request: can't parse inline keyboard button: Text buttons are unallowed in the inline keyboard

2023-04-13 17:17:52,988 DEBUG urllib3.connectionpool https://api.telegram.org:443 "GET /bot6234331500:AAG2Hcuep4y4N58uuJeXrHnjbpaSyrpXk3c/getUpdates?offset=379040608&timeout=20 HTTP/1.1" 200 23
2023-04-13 17:18:13,081 DEBUG urllib3.connectionpool https://api.telegram.org:443 "GET /bot6234331500:AAG2Hcuep4y4N58uuJeXrHnjbpaSyrpXk3c/getUpdates?offset=379040608&timeout=20 HTTP/1.1" 200 23
2023-04-13 17:18:33,594 DEBUG urllib3.connectionpool https://api.telegram.org:443 "GET /bot6234331500:AAG2Hcuep4y4N58uuJeXrHnjbpaSyrpXk3c/getUpdates?offset=379040608&timeout=20 HTTP/1.1" 200 23
2023-04-13 17:18:53,698 DEBUG urllib3.connectionpool https://api.telegram.org:443 "GET /bot6234331500:AAG2Hcuep4y4N58uuJeXrHnjbpaSyrpXk3c/getUpdates?offset=379040608&timeout=20 HTTP/1.1" 200 23
2023-04-13 17:19:13,797 DEBUG urllib3.connectionpool https://api.telegram.org:443 "GET /bot6234331500:AAG2Hcuep4y4N58uuJeXrHnjbpaSyrpXk3c/getUpdates?offset=379040608&timeout=20 HTTP/1.1" 200 23
2023-04-13 17:19:33,890 DEBUG urllib3.connectionpool https://api.telegram.org:443 "GET /bot6234331500:AAG2Hcuep4y4N58uuJeXrHnjbpaSyrpXk3c/getUpdates?offset=379040608&timeout=20 HTTP/1.1" 200 23
2023-04-13 17:19:39,902 ERROR TeleBot Infinity polling: polling exited
2023-04-13 17:19:39,902 ERROR TeleBot Break infinity polling
2023-04-13 17:19:39,903 INFO root Bot STOPPED
